scripts/release/prepare-release-from-ci.js

```diff
@@ -23,7 +23,8 @@ const run = async () => {
       await testPackagingFixture(params);
     }
 
-    await printPrereleaseSummary(params, false);
+    const isLatestRelease = params.releaseChannel === 'latest';
+    await printPrereleaseSummary(params, isLatestRelease);
```

Comment:
Why do we pin differently for the latest release?

Reply:
What's "pin" mean in this context?

Reply:
D'oh, nevermind sorry. I misread this.